CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.20)
+project(imgui LANGUAGES C CXX)
+
+add_library(imgui imgui.cpp imgui_draw.cpp imgui_tables.cpp imgui_widgets.cpp)
+target_include_directories(imgui PUBLIC .)
```

Comment:
Here and everywhere else `${CMAKE_CURRENT_LIST_DIR}` seems to be more explicit for me than a "."